### DISCUSSION TITLE
CMI tab group sorting  priority list

### DISCUSSION THREAD

**Discussion by D4nkM3mer** [Zrips/CMI#6343] [Enhancement]
Can you guys add group priority list in tab, that would be wonderful =)

**Comment by Strahilchu** [comment]
It's been asked for many times since a while ago, I even asked a few months ago. Doesn't look like its coming they just throw this request in some off-topic area that no one looks at

**Reply by Strahilchu** [reply]
Yep here it is in the discussions section that no one looks at

**Reply by D4nkM3mer** [reply]
Sad moment

**Comment by CloudeLecaw** [comment]
Yes optional priority would be nice ;D pls add

**Comment by D4nkM3mer** [comment]
Add it pls =)

**Comment by mrfloris** [comment]
Tablist.yml has player-group sorting in the latest builds of cmilib and cmi. Upgrade and review tablist.yml's configuration (bottom of page). You can sort ascending or descending by type, and types range from permission node weight to baltop rank, etc.
